hadoop-cloud/pom.xml

```diff
@@ -201,7 +201,7 @@
      enables store-specific committers.
     -->
     <profile>
-      <id>hadoop-3.2</id>
+      <id>hadoop-3</id>
```

Comment:
Hey, there was already WIP PT to replace this, see also https://github.com/apache/spark/pull/30891 cc @sunchao